pkg/component/networking/istio/istio_test.go

```diff
@@ -232,8 +232,18 @@ var _ = Describe("istiod", func() {
 			return string(data)
 		}
 
-		istioProxyProtocolEnvoyFilter = func() string {
-			data, _ := os.ReadFile("./test_charts/proxyprotocol_envoyfilter.yaml")
+		istioProxyProtocolEnvoyFilter8443 = func() string {
```

Comment:
Again, I would vote for semantic naming, i.e. not including the port names, but rather the name of the port.